mipengine/controller/api/services/validate_algorithm.py

```diff
@@ -75,7 +75,7 @@ def validate_inputdata(inputdata_specs: InputDataSpecificationsDTO,
     validate_inputdata_pathology_and_dataset_values(input_data.pathology,
                                                     input_data.datasets)
 
-    validate_inputdata_filter(input_data.filter)
+    validate_inputdata_filter(input_data.filters)
```

Comment:
Didn't we decide that we should call it filter? 

https://github.com/madgik/MIP-Engine/commit/d19cf716c9292b9be4888962ea3ed3345661478b

Don't remember what we decided TBH